appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: '0.0.{build}'
+version: '0.1.{build}'
 image: Visual Studio 2017
 init:
   # Good practise, because Windows line endings are different from Unix/Linux ones
```

Comment:
General notes, unrelated to this file:

Reply:
You use R#; so export your code settings and store them in your repository (so it can eventually be shared among all participating developers).

Reply:
You use R# locally, but build machine or some developers fighting it might not be so keen about their (in)ability to run tests - if you include NUnit test adapter among your tests' project's NuGet pacakges, even bare Visual Studio and most CI tools will be able to run them without any further setting…

Reply:
All your code is now dependant on `Unity`. If you decided to replace it with `CasteWindsor` or `SimpleInjector` or … You would have to replace it in all assemblies. Let's work on this concept further and isolate your DI provider from the rest of the solution… (I will preferably discuss this in person)